JitsiConference.js

```diff
@@ -371,13 +374,31 @@ JitsiConference.prototype._init = function(options = {}) {
     this.eventManager.setupStatisticsListeners();
 
     if (config.enableTalkWhileMuted) {
-        // eslint-disable-next-line no-new
-        new TalkMutedDetection(
-            this,
-            () =>
+        // If VAD processor factory method is provided uses VAD based detection, otherwise fallback to audio level
+        // based detection.
+        if (config.createVADProcessor) {
+            logger.info('Using VAD detection for generating talk while muted events');
+            // eslint-disable-next-line no-new
```

Comment:
I guess you don't need the eslint disable here?

Reply:
Noup, I forgot to remove some of the disables from when I was testing. Removed now.